app/views/layouts/application.html.erb

```diff
@@ -13,22 +13,26 @@
 
       <sticky>
         <%= render 'global/topbar' %>
-        
+
         <div class="header-wrapper" slot="header">
           <%= render 'global/header' %>
         </div>
-        
+
         <div slot="control-bar">
           <%= render 'global/control-bar' %>
         </div>
       </sticky>
-      
+
       <%= yield %>
 
     </div>
 
     <%= render 'global/footer' %>
 
     <%= javascript_pack_tag 'application' %>
+    <% if Rails.env == 'production' %>
```

Comment:
Can you please change this to include `staging` as well for the moment? So, after the merge, you could go to staging and test if it works correctly :)

Reply:
Sure @Levia I will do that now :)